### ISSUE TITLE
Timeline layout gets messy

### ISSUE THREAD

**Issue by espectalll** [tuskyapp/Tusky#1122]
For some reason, some toots in Tusky 5.2 are displaced and overlaying each other. I'll just show some screenshots, it's self-descriptive.

![](https://i.imgur.com/8oXPxRZ.png)
![](https://i.imgur.com/mPqIkCQ.png)
![](https://i.imgur.com/lZgUYEo.png)
![](https://i.imgur.com/fp82Q58.png)

**Comment by espectalll** [comment]
(in case you wonder: this is stock Android 9 Pie with February patches on a Sony Xperia XZ2 Compact – no custom ROM, no root)

**Comment by charlag** [comment]
I think I've seen this bug at least once. Does it reproduce reliably for you?

**Comment by connyduck** [comment]
I never had the bug on one of my devices, but saw screenshots. Would be very interested in how to reproduce this so I can fix it.

**Comment by espectalll** [comment]
I'm trying but still no luck, it's probably some rare race condition?

**Comment by charlag** [comment]
Shouldn't be race, rather a bug in one of the libraries. Let's see if it happens often.

**Comment by charlag** [comment]
@espectalll does this still happen to you? I didn't see this in a while

**Comment by espectalll** [comment]
@charlag yes, but it seems completely unpredictable, happens rarely and I can't properly test or debug it
